web/src/features/fba/components/infoPanel/AdvisoryText.tsx

```diff
@@ -41,6 +42,33 @@ const AdvisoryText = ({
     return 0
   }
 
+  // Return a list of fuel stats for which greater than 90% of the area of each fuel type has high HFI.
+  const getTopFuelsByProportion = (zoneUnitFuelStats: FireZoneFuelStats[]): FireZoneFuelStats[] => {
+    const topFuelsByProportion = zoneUnitFuelStats.filter(fuelStats => {
+      return fuelStats.area / fuelStats.fuel_area >= 0.9
+    })
+    return topFuelsByProportion
+  }
+
+  // Return a list of fuel types that cummulatively account for more than 75% of total area with high HFI.
+  const getTopFuelsByArea = (zoneUnitFuelStats: FireZoneFuelStats[]): FireZoneFuelStats[] => {
+    let totalHighHFIArea = 0
+    zoneUnitFuelStats.forEach(stats => {
+      totalHighHFIArea += stats.area
+    })
+    const topFuelsByArea = []
+    let highHFIArea = 0
+    for (const stats of zoneUnitFuelStats) {
```

Comment:
Should this block not be something like `getTopFuelsByProportion` above, just using 75% threshold and the high HFI area as the denominator? E.g.

    const topFuelsByArea = zoneUnitFuelStats.filter(fuelStats => {
      return fuelStats.area / totalHighHFIArea >= 0.75
    })
    return topFuelsByProportion

Reply:
I don't think so, though I'm trying to clarify the first acceptance criteria of #4195 with Neal. My interpretation is that we want to list the most prominent fuel types by area under advisory/warning such that the combined area of the primary fuel types account for >=75% of the total area under advisory. So the block of code you're questioning is concerned with cumulative area whereas the other block considers the proportion of individual fuel types.

Reply:
I see, thanks for clarifying, re-reading the AC and what question it answers makes sense.